src/app_builder.rs

```diff
@@ -294,7 +294,7 @@ where
         }
     }
 
-    /// Overwrites the default custom messages handler.
+    /// Overwrites the default custom messages' handler.
```

Comment:
Is this ' a typo?

Reply:
Changed to: *Overwrites the default handler for custom messages.*